DQMOffline/Trigger/python/VBFMETMonitor_cff.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.ObjMonitor_cfi import hltobjmonitoring
+
+# HLT_
+DiJetVBFmonitoring = hltobjmonitoring.clone()
+DiJetVBFmonitoring.FolderName = cms.string('HLT/Higgs/VBFMET/DiJet/')
+DiJetVBFmonitoring.numGenericTriggerEventPSet.hltInputTag   = cms.InputTag( "TriggerResults","","HLT" )
+DiJetVBFmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DiJet110_35_Mjj650_PFMET110_v*","HLT_DiJet110_35_Mjj650_PFMET120_v*","HLT_DiJet110_35_Mjj650_PFMET130_v*")
+#DiJetVBFmonitoring.metSelection = cms.string("")
+DiJetVBFmonitoring.jetSelection = cms.string("pt>40 & abs(eta)<4.7")
+EXO_VBF_METmonitoring.jetId = cms.string("loose")
```

Comment:
hi @amagnan - it looks like this line is causing the crashes - perhaps EXO_VBF_METmonitoring is meant to be DiJetVBFmonitoring?